core/src/main/kotlin/org/evomaster/core/problem/rest/service/fitness/AbstractRestFitness.kt

```diff
@@ -1156,6 +1171,26 @@ abstract class AbstractRestFitness : HttpWsFitness<RestIndividual>() {
         handleNotRecognizedAuthenticated(individual, actionResults, fv)
     }
 
+    private fun handleVulnerabilities(
+        individual: RestIndividual,
+        actionResults: List<ActionResult>,
+        fv: FitnessValue
+    ) {
+
+        individual.seeMainExecutableActions().forEach {
+            val ar = actionResults.find { r -> r.sourceLocalId == it.getLocalId() } as RestCallResult
+
+            if (ar != null && ar.getResultValue(HttpWsCallResult.VULNERABLE_SSRF)?.toBoolean() ?: false) {
+                val scenarioId = idMapper.handleLocalTarget(
+                    idMapper.getFaultDescriptiveId(DefinedFaultCategory.SSRF, it.getName())
+                )
+                fv.updateTarget(scenarioId, 1.0, it.positionAmongMainActions())
+
+                ar.addFault(DetectedFault(DefinedFaultCategory.SSRF, it.getName(), null))
```

Comment:
instead of `null`, as context here you could use the name of the parameter that leads to the SSRF. (or could just leave it as a TODO comment)